code/modules/cm_preds/yaut_shield.dm

```diff
@@ -45,10 +45,10 @@
 		M.apply_effect(3, DAZE)
 		M.apply_effect(5, SLOW)
 
-/obj/item/weapon/shield/riot/yautja/attackby(obj/item/I, mob/user)
+/obj/item/weapon/shield/riot/yautja/attackby(obj/item/object, mob/user)
```

Comment:
For future reference yes, this is an object, but something like "attacking_item" would describe so much more and give clarification on explicit typepath rather than the parent.